### PR TITLE
Feat: add progressbar while page loading

### LINKED ISSUE #25: [P0] Show progress bar on page loads, refreshes, and button clicked
Check page loads with API or something else to show progress bar by the rate of rendered components.

### PR DESCRIPTION
**Pull request by sanghyun1128**
Use [Pace.js](https://github.com/CodeByZach/pace/) library.

This PR closes #25

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "@types/node": "^16.18.12",
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
+    "pace": "^0.0.4",
```
Comment (seiwonpark) [inline web/package.json]: What is this `pace` library for?

**Review by ohsung1125 — APPROVED** [review]
LGTM!

File: web/src/index.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
+import 'pace-js';
```
Comment (seiwonpark) [inline web/src/index.tsx]: You don't need this line since js bundle is already included in `index.html`

File: web/public/index.html [hunk 3]
```diff
@@ -28,6 +28,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>the survey</title>
+    <script src="https://unpkg.com/pace-js@1.2.4/pace.min.js"></script>
```
Comment (seiwonpark) [inline web/public/index.html]: For every ajax request, it doesn't have to show all restarted progress bar on every ajax update. So disable with `data-pace-options='{ "restartOnRequestAfter": false }'` property on this script. See https://github.com/CodeByZach/pace#configuration.

And also Youtube doesn't show every restarted progress bar on every ajax udpate.

**Review by seiwonpark — CHANGES_REQUESTED** [review]
Please update your code and re-request!

File: web/package.json (hunk 1 above)
Comment (sanghyun1128) [reply web/package.json]: pace is for progress bar in cmd. We don't use it, but I added it by mistake. So I deleted it.

**Review by seiwonpark — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**